### DISCUSSION TITLE
Add pdf viewing feature

### DISCUSSION THREAD

**Discussion by devalllex** [Acode-Foundation/Acode#644] [Ideas]
Please add fuction to view pdf in acode

**Comment by deadlyjack** [comment]
Sure, thank you for the suggestion.

**Comment by atikattar1104** [comment]
I Would Suggest We Can Have Code From MJPDF App. It Is Open Source. Working Can Be Like :- 

1. Whenever A PDF File Opens, App Will Open It To View In Preview Section. 
2. Binary Data Can Be Shown In Default Coding Canvas, For Finding Issues In PDF Files.

I Think This Would Be A Great Feature.

**Reply by sebastianjnuwu** [reply]
I don't know in a code editing application? View PDF? In my opinion, it could do the same when opening an image, I can try to make a plugin that does this, I just need the example code example that opens images
